docs/whatsnew/5.2.rst

```diff
@@ -78,6 +78,12 @@ easier and more intuitive.::
     >>> # convert to AltAz
     >>> aa = itrs_topo.transform_to(AltAz(obstime=t, location=home))
 
+Performance Improvements
+========================
+
+Several performance improvements were implemented, mainly concerning
+``Angle`` and ``CoordinateAttribute``.
```

Comment:
I feel this gives the user very little sense of what this is good for - I'd mention *think* you started this all because coordinate transformations were quite sluggish. Maybe "To help speed up coordinate transformations, several ..."?

Reply:
done